### DISCUSSION TITLE
New certificate (godaddy DNS) issues but "Add Certificate" page never returns

### DISCUSSION THREAD

**Discussion by BarryBahrami** [polymind-inc/acmebot#731] [Q&A]
I have setup this great software in Azure, using GoDaddy DNS.  It is issuing a certificate just fine but when I go through the "Add Certificate" dialogue to issue a new certificate, the green box just continues spinning. It never actually returns success.  I have to refresh the browser and I see it listed as normal.

I have this working on multiple other instances, although that is with Azure DNS.  Can't figure out why this isn't returning success.   Has anyone experienced this before?

Thank you in advance

**Comment by webprofusion-chrisc** [comment]
Note that GoDaddy recently changed their API so that only customers with 10+ domains on GoDaddy can use it for DNS operations. It still  may work for certain primary domain operations but not subdomains.
